enterprise/server/remote_execution/containers/podman/podman.go

```diff
@@ -287,3 +296,31 @@ func killContainerIfRunning(ctx context.Context, containerName string) error {
 	}
 	return status.UnknownErrorf("podman kill failed: %s", string(result.Stderr))
 }
+
+// An image can be corrupted if "podman pull" command is killed when pulling a parent layer.
+// More details can be found at https://github.com/containers/storage/issues/1136. When this
+// happens when need to remove the image before re-pulling the image in order to fix it.
+func (c *podmanCommandContainer) maybeCleanupCorruptedImages(ctx context.Context, result *interfaces.CommandResult) error {
+	if result.ExitCode != podmanInternalExitCode {
+		return nil
+	}
+	if !storageErrorRegex.MatchString(string(result.Stderr)) {
+		return nil
+	}
+	result.Error = status.UnavailableError("a storage corruption occurred for podman")
```

Comment:
Set `result.ExitCode = commandutil.NoExitCode` here (the convention for CommandResult is that `ExitCode>=0` if and only if `Error==nil`)

Reply:
Done. Thanks for the explanation!